micromasters/urls.py

```diff
@@ -37,8 +43,11 @@
     url(r'^api/v0/search/(?P<elastic_url>.*)', ElasticProxyView.as_view(), name='search_api'),
     url(r'^api/v0/checkout/$', CheckoutView.as_view(), name='checkout'),
     url(r'^api/v0/enrolledprograms/$', ProgramEnrollmentListView.as_view(), name='user_program_enrollments'),
-    url(r'^api/v0/mail/$', MailView.as_view(), name='mail_api'),
-    url(r'^api/v0/financialaid/$', IncomeValidationView.as_view(), name='financialaid_api'),
+    url(r'^api/v0/search_result_mail/$', SearchResultMailView.as_view(), name='search_result_mail_api'),
```

Comment:
Please change back to `api/v0/mail`. This PR hasn't updated the JS client for the changed endpoint, and in any case I'd like to not rename things outside the scope of this PR if possible